src/util/metadataStatic_api_v1_17_2.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright 2017-2020 @polkadot/metadata authors & contributors
```

Comment:
For other people reading this file in the future, I would add a comment on this file, to let them know what's special about this metadata.

Also, metadata are uniquely defined by chainName + specVersion, so this file could be called: `metadataStatic_substrate_{n}.ts`

Reply:
See new comment at top of file.